.github/workflows/build-vcpkg.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       matrix:
         name: [ubuntu, ubuntu-clang, macos-clang, windows-msvc]
```

Comment:
I think we should get rid of `ubuntu-clang` here. Doesn't cover anything (MacOS already builds with clang) and adds 3 more builds that slow the workflow. cc @DSpeichert

Reply:
OK